docs/reference/ml/df-analytics/apis/explain-dfanalytics.asciidoc

```diff
@@ -45,9 +45,10 @@ The following explanations are provided:
 
 * which fields are included or not in the analysis and why,
 * how much memory is estimated to be required. The estimate can be used when 
-  deciding the appropriate value for `model_memory_limit` setting later on,
+  deciding the appropriate value for `model_memory_limit` setting later on.
 
-about either an existing {dfanalytics-job} or one that has not been created yet.
```

Comment:
This was stated twice in this paragraph so I took the chance to clean it up here.